discord/client.py

```diff
@@ -2053,9 +2053,7 @@ async def fetch_entitlements(
         guild: Snowflake | None = None,
         exclude_ended: bool = False,
     ) -> EntitlementIterator:
-        """|coro|
-
-        Fetches the bot's entitlements.
+        """Returns an :class:`.AsyncIterator` that enables fetching the application's entitlements.
 
         .. versionadded:: 2.5
```

Comment:
This should technically say 2.6 now that it's been renamed.

Reply:
Ah... i was completely unaware that this was previously in 2.5, making this breaking. Should the original name perhaps be included as a deprecated function, or leave it as is?